packages/ui-token-scripts/lib/commands/generate-tokens.js

```diff
@@ -22,81 +22,80 @@
+      type: 'string',
+      describe: 'Path to the JS source tokens for a theme.',
+      requiresArg: true
+    })
 
-  yargs.option('outputPackage', {
-    alias: 'p',
-    type: 'string',
-    describe: 'Package to output generated cross-platform tokens.',
-    requiresArg: true
-  })
+    yargs.option('themeKey', {
+      alias: 't',
+      type: 'string',
+      describe:
+        'Unique key to identify the theme, used in output path and token namespacing.',
+      requiresArg: true
+    })
 
-  yargs.option('outputDir', {
-    alias: 'o',
-    type: 'string',
-    describe:
-      'Output directory for generated tokens within the source theme package.',
-    default: 'tokens'
-  })
+    yargs.option('outputPackage', {
+      alias: 'p',
+      type: 'string',
+      describe: 'Package to output generated cross-platform tokens.',
+      requiresArg: true
+    })
 
-  yargs.option('groupOutput', {
-    alias: 'g',
-    type: 'boolean',
-    describe:
-      'Should generated tokens be grouped by theme in the output directory?',
-    default: false
-  })
-}
+    yargs.option('outputDir', {
+      alias: 'o',
+      type: 'string',
+      describe:
+        'Output directory for generated tokens within the source theme package.',
+      default: 'tokens'
+    })
 
-exports.handler = (argv) => {
-  const { sourceTokens, themeKey, outputPackage, outputDir, groupOutput } = argv
+    yargs.option('groupOutput', {
+      alias: 'g',
+      type: 'boolean',
+      describe:
+        'Should generated tokens be grouped by theme in the output directory?',
+      default: false
+    })
+  },
+  handler: async (argv) => {
+    const { sourceTokens, themeKey, outputPackage, outputDir, groupOutput } =
+      argv
 
-  const tokens = require(sourceTokens).default
+    const tokens = require(sourceTokens).default
 
-  if (Object.keys(tokens).indexOf('colors') < 0) {
-    error('Invalid token source')
-    process.exit(1)
-  }
+    if (Object.keys(tokens).indexOf('colors') < 0) {
+      error('Invalid token source')
+      process.exit(1)
+    }
 
-  const styleDictionarySource = handleMapJsTokensToSource(tokens)
-  const themePath = path.dirname(
-    require.resolve(path.join(outputPackage, 'package.json'))
-  )
-  const outputPath = groupOutput
-    ? path.join(themePath, outputDir, themeKey)
-    : path.join(themePath, outputDir)
-  const sourcePath = path.join(outputPath, 'source.json')
+    const styleDictionarySource = handleMapJSTokensToSource(tokens)
+    const themePath = path.dirname(
+      require.resolve(path.join(outputPackage, 'package.json'))
+    )
+    const outputPath = groupOutput
+      ? path.join(themePath, outputDir, themeKey)
+      : path.join(themePath, outputDir)
+    const sourcePath = path.join(outputPath, 'source.json')
 
-  try {
-    handleGenerateTokens({
+    await handleGenerateTokens({
       themeKey,
       styleDictionarySource,
       outputPath,
       sourcePath
     })
-  } catch (e) {
-    error(e)
```

Comment:
removing these catchall handlers fixes the bug that this script failing was not failing the whole build